W-14089647 streamline tuts ga v3 duke (#174)
mulesoft/docs-code-builder: modules/ROOT/pages/index.adoc

== Get Familiar with the Basics

> It's helpful to have some knowledge about designing an API using API specification languages (OAS, RAML, or GraphQL). API implementations and integration development require familiarity with Mule application development, connectors, and the DataWeave transformation language. Anypoint Code Builder xref:tut-af-deploy-am-flights.adoc[tutorials] help you get started with these tasks.

Reviewer: ```suggestion
Some knowledge about designing an API using API specification languages (OAS, RAML, or GraphQL) can be helpful. API implementations and integration development require familiarity with Mule application development, connectors, and the DataWeave transformation language. Anypoint Code Builder xref:tut-af-deploy-am-flights.adoc[tutorials] help you get started with these tasks.
```

Reviewer: Try to avoid "It is" construction.
